includes/queue/class-wc-action-queue.php

```diff
@@ -126,11 +126,11 @@ public function get_next( $hook, $args = null, $group = '' ) {
 
 		$next_timestamp = as_next_scheduled_action( $hook, $args, $group );
 
-		if ( $next_timestamp ) {
-			return new WC_DateTime( "@{$next_timestamp}", new DateTimeZone( 'UTC' ) );
+		if ( true === $next_timestamp || false === $next_timestamp ) {
```

Comment:
I just checked the functions docblock in: 

https://github.com/woocommerce/action-scheduler/blob/f3a13eb9b5c9b1f06db1144caa219202c27deef6/functions.php#L149-L164

Sounds like that we could just change to:

```suggestion
		if ( is_numeric( $next_timestamp ) ) {
```

Reply:
Okay, that makes sense, except we want the opposite here which should be if `is_bool`

Reply:
Oh, nevermind.. i see your full change now, you swapped the order... github broke it up funny so it was hard to see.